README.md

```diff
@@ -292,7 +292,7 @@ mContext.acquireTokenSilent(String resource, String clientId, String userId, fin
 	* If target version is lower than 23, calling app has to have the following permissions declared in manifest(http://developer.android.com/reference/android/accounts/AccountManager.html):
 		* GET_ACCOUNTS
 		* USE_CREDENTIALS
-		* MANAGE_ACOUNTS
+		* MANAGE_ACCOUNTS
```

Comment:
That's a change already checked into 'dev' branch by the author. It appeared in the "New changes since you last viewed" section in this PR, but it's not part of my change. It won't affect my PR.